[2024-05-06T07:26:53.634+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-06T07:26:53.880+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tweets_dag_v4.init_tweets_db manual__2024-05-06T06:45:39.910403+00:00 [queued]>
[2024-05-06T07:26:53.912+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tweets_dag_v4.init_tweets_db manual__2024-05-06T06:45:39.910403+00:00 [queued]>
[2024-05-06T07:26:54.014+0000] {taskinstance.py:2303} INFO - Starting attempt 12 of 17
[2024-05-06T07:26:54.210+0000] {taskinstance.py:2327} INFO - Executing <Task(PostgresOperator): init_tweets_db> on 2024-05-06 06:45:39.910403+00:00
[2024-05-06T07:26:54.257+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tweets_dag_v4', 'init_tweets_db', 'manual__2024-05-06T06:45:39.910403+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/twitter_data_dag.py', '--cfg-path', '/tmp/tmpdu945j34']
[2024-05-06T07:26:54.278+0000] {standard_task_runner.py:91} INFO - Job 112: Subtask init_tweets_db
[2024-05-06T07:26:54.274+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=447) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-06T07:26:54.287+0000] {standard_task_runner.py:63} INFO - Started process 591 to run task
[2024-05-06T07:26:54.551+0000] {task_command.py:426} INFO - Running <TaskInstance: tweets_dag_v4.init_tweets_db manual__2024-05-06T06:45:39.910403+00:00 [running]> on host 5efa3aaee11b
[2024-05-06T07:26:55.060+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stan' AIRFLOW_CTX_DAG_ID='tweets_dag_v4' AIRFLOW_CTX_TASK_ID='init_tweets_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T06:45:39.910403+00:00' AIRFLOW_CTX_TRY_NUMBER='12' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T06:45:39.910403+00:00'
[2024-05-06T07:26:55.080+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-06T07:26:55.146+0000] {sql.py:276} INFO - Executing: CREATE TABLE IF NOT EXISTS positive_tweets (
    id SERIAL PRIMARY KEY,
    roberta_neg NUMERIC,
    roberta_neu NUMERIC,
    roberta_pos NUMERIC,
    author VARCHAR(50),
    content TEXT,
    date_time TIMESTAMP,
    number_of_likes INTEGER,
    number_of_shares INTEGER
);

CREATE TABLE IF NOT EXISTS negative_tweets (
    id SERIAL PRIMARY KEY,
    roberta_neg NUMERIC,
    roberta_neu NUMERIC,
    roberta_pos NUMERIC,
    author VARCHAR(50),
    content TEXT,
    date_time TIMESTAMP,
    number_of_likes INTEGER,
    number_of_shares INTEGER
);

CREATE TABLE IF NOT EXISTS neutral_tweets (
    id SERIAL PRIMARY KEY,
    roberta_neg NUMERIC,
    roberta_neu NUMERIC,
    roberta_pos NUMERIC,
    author VARCHAR(50),
    content TEXT,
    date_time TIMESTAMP,
    number_of_likes INTEGER,
    number_of_shares INTEGER
);

TRUNCATE TABLE positive_tweets;
TRUNCATE TABLE negative_tweets;
TRUNCATE TABLE neutral_tweets;

[2024-05-06T07:26:55.195+0000] {base.py:84} INFO - Using connection ID 'postgres_database' for task execution.
[2024-05-06T07:26:55.261+0000] {base.py:84} INFO - Using connection ID 'postgres_database' for task execution.
[2024-05-06T07:26:55.320+0000] {sql.py:457} INFO - Running statement: CREATE TABLE IF NOT EXISTS positive_tweets (
    id SERIAL PRIMARY KEY,
    roberta_neg NUMERIC,
    roberta_neu NUMERIC,
    roberta_pos NUMERIC,
    author VARCHAR(50),
    content TEXT,
    date_time TIMESTAMP,
    number_of_likes INTEGER,
    number_of_shares INTEGER
);

CREATE TABLE IF NOT EXISTS negative_tweets (
    id SERIAL PRIMARY KEY,
    roberta_neg NUMERIC,
    roberta_neu NUMERIC,
    roberta_pos NUMERIC,
    author VARCHAR(50),
    content TEXT,
    date_time TIMESTAMP,
    number_of_likes INTEGER,
    number_of_shares INTEGER
);

CREATE TABLE IF NOT EXISTS neutral_tweets (
    id SERIAL PRIMARY KEY,
    roberta_neg NUMERIC,
    roberta_neu NUMERIC,
    roberta_pos NUMERIC,
    author VARCHAR(50),
    content TEXT,
    date_time TIMESTAMP,
    number_of_likes INTEGER,
    number_of_shares INTEGER
);

TRUNCATE TABLE positive_tweets;
TRUNCATE TABLE negative_tweets;
TRUNCATE TABLE neutral_tweets;
, parameters: None
[2024-05-06T07:26:55.491+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-06T07:26:55.498+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 282, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 404, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 462, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "pg_type_typname_nsp_index"
DETAIL:  Key (typname, typnamespace)=(positive_tweets_id_seq, 2200) already exists.

[2024-05-06T07:26:55.555+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=tweets_dag_v4, task_id=init_tweets_db, execution_date=20240506T064539, start_date=20240506T072653, end_date=20240506T072655
[2024-05-06T07:26:55.600+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 112 for task init_tweets_db (duplicate key value violates unique constraint "pg_type_typname_nsp_index"
DETAIL:  Key (typname, typnamespace)=(positive_tweets_id_seq, 2200) already exists.
; 591)
[2024-05-06T07:26:55.665+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-06T07:26:55.790+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1296 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-05-06T07:26:55.845+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-06T07:26:55.857+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
